src/python/grpcio/grpc/__init__.py

```diff
@@ -348,6 +348,82 @@ def details(self):
         raise NotImplementedError()
 
 
+##############  Invocation-Side Interceptor Interfaces & Classes  ##############
+
+
+class UnaryClientInfo(six.with_metaclass(abc.ABCMeta)):
```

Comment:
I can make that change. But one advantage of doing it this way is that it's easy to support adding additional data in the future without breaking existing interceptors.